Lib/dataclasses.py

```diff
@@ -992,6 +993,10 @@ def _process_class(cls, init, repr, eq, order, unsafe_hash, frozen):
         cls.__doc__ = (cls.__name__ +
                        str(inspect.signature(cls)).replace(' -> None', ''))
 
+    # update the abstract methods of the class, if it is abstract
```

Comment:
I know this is nitpicking, but all of the other comments in this file start with a capital letter and end in a period.